### ISSUE TITLE
Unhandled characters in build info arg

### ISSUE THREAD

**Issue by Puyodead1** [camden-smallwood/pdb-decompiler#16]
```
thread 'main' panicked at 'Unhandled characters in build info arg: 'Brepro'', src\cpp\module.rs:530:32
stack backtrace:
   0: std::panicking::begin_panic_handler
             at /rustc/2c8cc343237b8f7d5a3c3703e3a87f2eb2c54a74/library\std\src\panicking.rs:575
   1: core::panicking::panic_fmt
             at /rustc/2c8cc343237b8f7d5a3c3703e3a87f2eb2c54a74/library\core\src\panicking.rs:64
   2: pdb_decompiler::cpp::module::Module::add_build_info
             at .\src\cpp\module.rs:530
   3: pdb_decompiler::process_id_information
             at .\src\main.rs:188
   4: pdb_decompiler::decompile_pdb
             at .\src\main.rs:108
   5: pdb_decompiler::main
             at .\src\main.rs:76
   6: core::ops::function::FnOnce::call_once<enum2$<core::result::Result<tuple$<>,alloc::boxed::Box<dyn$<core::error::Error>,alloc::alloc::Global> > > (*)(),tuple$<> >
             at /rustc/2c8cc343237b8f7d5a3c3703e3a87f2eb2c54a74\library\core\src\ops\function.rs:250```

**Comment by camden-smallwood** [comment]
Hello, can you share a link to the PDB file by chance? If so, I can debug it locally to see where the issue resides. If not, you can always try commenting out [these lines](https://github.com/camden-smallwood/pdb-decompiler/blob/bb77f54c36adb17ecf7bd91e8ed0039e173a0d2e/src/cpp/module.rs#L487-L1150) to disable parsing the command line options.

**Comment by camden-smallwood** [comment]
I found the issue, `/Brepro` was an undocumented MSVC command line option, which it wasn't implemented. This has been implemented as of commit [2831bc1](https://github.com/camden-smallwood/pdb-decompiler/commit/2831bc1a092a9c035080cf7bfcd99155c1e0a6d7).

**Comment by Puyodead1** [comment]
Sorry, I missed your first comment. 

Thank you!

**Comment by Puyodead1** [comment]
There's actually a lot more, so I'll just upload the PDB in case you want to implement them.
https://pixeldrain.com/u/bv3Mx3Kx stellaris.pdb

**Comment by camden-smallwood** [comment]
Thanks for sharing, I'll re-open the issue, run through them and send an update after getting them implemented.

**Comment by camden-smallwood** [comment]
These should all be implemented as of commit [528609d](https://github.com/camden-smallwood/pdb-decompiler/commit/528609de7b7074349765a81480779f928abbc96f) now.
